CONTRIBUTING.md

```diff
@@ -169,17 +169,14 @@ During development
 
 To perform a release of next `<version>`:
 
-1. Check version to be released is also correct in software components, e.g. `.cabal` files.
-2. Refer to `<version>` in deployment instructions, e.g. `docker-compose.yaml` files.
-3. Replace `UNRELEASED` with a date in [ISO8601](https://en.wikipedia.org/wiki/ISO_8601)
-4. Publish hydra scripts onto `preview`/`pre-prod` using the [smoke
+1. Publish hydra scripts onto `preview`/`pre-prod` using the [smoke
    test](https://github.com/input-output-hk/hydra/actions/workflows/smoke-test.yaml)
    and update transaction ids in our documentation, examples and
    `testnets/<network/hydra-scripts-tx-id` files. This will also be the green
    light for the next step.
-5. Create a signed, annotated git tag of the version: `git tag -as <version>`
-   - Copy/paste this release CHANGELOG as annotation
-6. Create a github release page containing
+2. Replace `UNRELEASED` with a date in [ISO8601](https://en.wikipedia.org/wiki/ISO_8601) in CHANGELOG.md
+3. run `./release` with `<version>` and the just published `hydra-scripts-tx-id` from step 1
```

Comment:
I suggested renaming release to `release.sh` to be consistent. I'm happy to change the others if you prefer that instead. In any case, don't forget to update this here if we change it.